### PR TITLE
feat(changeHeight): allow apester units to resize

### PR DESCRIPTION
**Pull request by mrsufgi**
This is a clean PR of the original https://github.com/ampproject/amphtml/pull/4755 PR.

@dvoytenko, this also fix the test issue. I hope we're good here. 

We can use this PR to debate on how to address 'unknown height' with playlist mode. 

Best,
Ori

@OmriKeret

### CODE REVIEW

**Comment by mrsufgi** [comment]
@googlebot CLA was sign by Apester LTD

**Comment by mrsufgi** [comment]
@dvoytenko Hey, Its been awhile. how are we doing on this? 

I don't know why @googlebot  misbehaves :)

**Comment by dvoytenko** [comment]
@mrsufgi Sorry, I confused this PR with the other one that had bunch of merge problems. Merging...

**Comment by mrsufgi** [comment]
Thank you! 

With both PR merged. Could we get apester-media out of experimental? This needs a separate PR?

File: extensions/amp-apester-media/0.1/amp-apester-media.js [hunk 1]
```diff
@@ -206,7 +206,11 @@ class AmpApesterMedia extends AMP.BaseElement {
           const height = 0 || media.data.size.height;
           if (height != this.height_) {
             this.height_ = height;
-            this./*OK*/attemptChangeHeight(height);
+            if (this.random_) {
+              this./*OK*/attemptChangeHeight(height);
+            } else {
+              this./*OK*/changeHeight(height);
```
Comment (dvoytenko) [inline extensions/amp-apester-media/0.1/amp-apester-media.js]: Correct. This is in response to Apester calculating the right height when embed block is generated.

**Pull request merged**